docs/modules/deployment/pages/cloud-platforms.adoc

```diff
@@ -55,9 +55,11 @@ eksctl create iamserviceaccount \
 
 ==== Step 4: Deploy Cerbos with the service account
 
+Firstly create your Cerbos Hub account and connect it to your policy store - see the xref:cerbos-hub:ROOT:getting-started.adoc[Cerbos Hub getting started guide] for details.
```

Comment:
Isn't this redundant because the banner below tells the user it's a Hub feature.

Reply:
The banner isn't overly descriptive in context nor includes the link to the guide for getting started with your account

![image](https://github.com/user-attachments/assets/652f50d3-c7cb-4520-b6d2-ba18dee56043)

Reply:
OK. Let's move it below the banner and re-word it as follows.

```suggestion
For the following steps, you need a Cerbos Hub account with a workspace connected to your policy repository and a set of client credentials. See the xref:cerbos-hub:ROOT:getting-started.adoc[Cerbos Hub getting started guide] for details.
```